skopt/optimizer/base.py

```diff
@@ -277,18 +278,25 @@ def base_minimize(func, dimensions, base_estimator,
 
         models.append(gp)
 
-        if acq_optimizer == "sampling":
+        if acq_optimizer == "sampling" or acq_optimizer == "lbfgs_optim":
```

Comment:
It might be better to implement this in "lbfgs" itself rather than create a separate acquisition function.